app/src/main.c

```diff
@@ -319,5 +319,10 @@ int main(int argc, char *argv[]) {
 
     avformat_network_deinit(); // ignore failure
 
+#if defined (__WINDOWS__) && ! defined (WINDOWS_NOCONSOLE)
+    if (res != 0) {
+        system("pause");
```

Comment:
I'm ok with the idea to let the window open on Windows, but more with something like `getchar()` (I cannot test currently, I have no Windows). Calling `system()` should be avoided when possible (e.g. what happens if a `pause.exe` if is available in `PATH`? etc.).

Reply:
(if you can test with `getchar()` on Windows, please open a new PR, I'm closing this one since the main change has been merged, and this one is somewhat unrelated)